src/goto-symex/slice.cpp

```diff
@@ -170,3 +193,15 @@ BigInt simple_slice(std::shared_ptr<symex_target_equationt> &eq)
 
   return ignored;
 }
+
+BigInt slice(std::shared_ptr<symex_target_equationt> &eq)
```

Comment:
Add some comments here.

Reply:
I've put the comments in the header file, OK?